### PR TITLE
NA label custom msg

### PR DESCRIPTION
**Pull request by karima662**
edited `apply_row_grp_lbl` to:

- store label column values
- check if any NAs present
- output custom message if true

### CODE REVIEW

File: R/apply_row_grp_plan.R [hunk 1]
```diff
@@ -96,6 +96,15 @@ apply_row_grp_struct <- function(.data, row_grp_struct_list, group, label = NULL
 #' @importFrom rlang !!!
 apply_row_grp_lbl <- function(.data, element_row_grp_loc, group, label = NULL, ...){
 
+  # store values of label column
+  lbl_col <- rlang::eval_tidy(label, .data)
```
Comment (bzkrouse) [inline R/apply_row_grp_plan.R:100]: please move this to @importFrom rlang eval_tidy, and remove the rlang::

**Review by bzkrouse — CHANGES_REQUESTED** [review]
This is just what we need thank you!! Just a tiny change to do and we should be good!

File: R/apply_row_grp_plan.R (hunk 1 above)
Comment (karima662) [reply R/apply_row_grp_plan.R]: see latest commit :)

**Review by bzkrouse — APPROVED** [review]
thank you!!

**Pull request merged**